study-builder/studies/atcp/snippets/question/text/text-question-email.conf

```diff
@@ -0,0 +1,56 @@
+{
+  include required("../../../../snippets/text-question.conf"),
```

Comment:
If we remove the `stay_informed` activity, we can then remove these email/firstname/lastname snippets.